tests/integration/cartography/intel/aws/test_s3_acls.py

```diff
@@ -0,0 +1,51 @@
+from cartography.intel.aws.s3 import _load_s3_acls
+from cartography.intel.aws.s3 import load_s3_buckets
+from cartography.intel.aws.s3 import parse_acl
+from tests.data.aws.s3 import LIST_BUCKETS
+from tests.data.aws.s3 import OPEN_BUCKET_ACLS
+from tests.integration.cartography.intel.aws.iam.test_iam import _create_base_account
+from tests.integration.util import check_nodes
+
+TEST_ACCOUNT_ID = '000000000000'
+TEST_REGION = 'us-east-1'
+TEST_UPDATE_TAG = 123456789
+
+
+def test_load_s3_acls(neo4j_session):
+    """
+    Ensure that the S3 ACL that sets anonymous_access and anonymous_actions works.
+    """
+    # Arrange: hackily add s3 data to the test graph
+    _create_base_account(neo4j_session)
+    load_s3_buckets(neo4j_session, LIST_BUCKETS, TEST_ACCOUNT_ID, TEST_UPDATE_TAG)
+    parsed_acls = []
+    for bucket_name, acl in OPEN_BUCKET_ACLS.items():
+        acl = parse_acl(acl, bucket_name, TEST_ACCOUNT_ID)
+        if acl:
+            parsed_acls.extend(acl)
+
+    # Act: run the analysis job
+    _load_s3_acls(neo4j_session, parsed_acls, TEST_ACCOUNT_ID, TEST_UPDATE_TAG)
+
+    # Assert that the anonymous_access field is set as expected
+    assert check_nodes(neo4j_session, 'S3Bucket', ['name', 'anonymous_access']) == {
+        ('bucket-1', None),
+        ('bucket-3', True),
+        ('bucket-2', True),
+    }
+
+    # Assert that we properly set anonymous_actions on the S3Bucket based on the attached S3Acls
+    # (Can't use check_nodes() here because anonymous_actions is a list and is non-hashable.)
```

Comment:
non-blocker: A few weeks ago we had a _check_nodes_as_list in the semgrep test. We could refactor it into tests.integraton.util
https://github.com/lyft/cartography/blob/ef5cbcede534d91df74d2e497cc2958ae548271e/tests/integration/cartography/intel/semgrep/test_findings.py#L19